examples/step-71/doc/intro.dox

```diff
@@ -209,15 +209,15 @@ for an equation that has two derivatives on each shape function. In
 analogy to the Interior Penalty (IP) method for the Laplace equation,
 this scheme for the biharmonic equation is typically called the $C^0$ IP
 (or C0IP) method, since it uses $C^0$ (continuous but not continuously
-differentiable) shape functions with an interior penalty.
+differentiable) shape functions with an interior penalty formulation.
 
 
 <h3> Derivation of the $C^0$ IP method </h3>
 
 We base this program on the $C^0$ IP method presented by Susanne
 Brenner and Li-Yeng Sung in the paper "C$^0$ Interior Penalty Method
 for Linear Fourth Order Boundary Value Problems on polygonal
-domains'' @cite Brenner2005 , where the method is
+domains'' (@cite Brenner2005), where the method is
```

Comment:
That's going to look awkward because it puts round parentheses around the square brackets.